enterprise-suite/values.yaml

```diff
@@ -75,6 +75,16 @@ imageCredentials:
 #  username: setme
 #  password: setme
 
+# If true, an alertmanager will be deployed. Set to false and modify `alertManagers` list to use existing alertmanager(s).
+createAlertManager: true
+
+# List of alertmanager addresses (can be k8s local DNS service names)
+alertManagers: 
+  - alertmanager
```

Comment:
I'm not sure this should be a list. Doesn't that make it awkward to specify on the command line? I think a single var that is comma separated would be fine. Since this gets passed directly to the es-monitor-api as a command line option.

Reply:
Making it a list has advantage of easier manipulation in template logic. For example, I need to get first item for setting up nginx forwarding. Also, I feel it's more idomatic since YAML has list data structure? That makes it nicer if you want to specify values in a YAML file instead of passing everything as commandline arguments.

Commandline looks like this now:
`--set alertManagers={alertmanager-00,alertmanager-01,alertmanager-02}`
vs. comma separated list:
`--set alertManagers=alertmanager-00,alertmanager-01,alertmanager-02`